### PR TITLE
Person - Documentation

### PR DESCRIPTION
**Pull request by cerbs**
Documentation for the Person resource

Note:
- There are no terminology bindings or implementation notes since no non-required terminologies supported and no ignored modifying elements

- Built from the same base as https://github.com/cerner/fhir.cerner.com/pull/84, so this includes changes to break apart the example json into individual files per resources

### CODE REVIEW

**Comment by daliboz** [comment]
We should add binding for the Identifier type

**Comment by daliboz** [comment]
Otherwise +1

File: content/dstu2/person.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Search for Persons that meet supplied query parameters:
+
+    GET /Person?:parameters
+
+### Parameters
+
+ Name         | Required?                                                    | Type       | Description
+--------------|--------------------------------------------------------------|------------|------------------------------------------------------------------------------------
+ `_id`        | No, if populated all other parameters are ignored            | [`token`]  | The logical resource id associated with the resource.
+ `identifier` | Yes, or `_id`                                                | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```
Comment (MaxPhilips) [inline content/dstu2/person.md]: Could trim out some of the empty space in this table to keep the style consistent with other resources :)

```
 Name         | Required?                                         | Type      | Description
--------------|---------------------------------------------------|-----------|-------------------------------------------------------
 `_id`        | No, if populated all other parameters are ignored | [`token`] | The logical resource id associated with the resource.
 `identifier` | Yes, or `_id`                                     | [`token`] | The person identifier.  Example: `01022228`
```

**Comment by MaxPhilips** [comment]
+1

**Comment by cerbs** [comment]
Merged and squashed on commit 2e210ff

**Pull request closed without merging**